packages/react-native/lib/default-biometrics-plugin.ts

```diff
@@ -185,7 +185,8 @@ const issueBiometricMatchCredential = async enrollmentCredential => {
   let created = getIssuanceDate();
 
   if (schema?.id) {
-    const fetchSchemaResponse = await axios.get(schema?.id).catch((err) => {
+    // React native iOS can't handle cross-origin redirects
+    const fetchSchemaResponse = await axios.get(schema?.id?.replace('dock', 'truvera')).catch((err) => {
```

Comment:
should we be more specific to avoid accidentally replacing it in the wrong spot? i.e. replace on schema.dock.io

Reply:
good point, let me update that logic